packages/adsum-utils/services/ClientAPI.js

```diff
@@ -123,6 +123,19 @@ class ClientAPI {
         return this.entityManager.getRepository('Category').findBy(filter);
     }
 
+    getCategoriesByTag(tagName) {
+        let categoriesToReturn = [];
+        const tags = this.getTagBy({name: tagName});
+
+        for (let tag of tags) {
+            for (let category of tag.categories.values) {
+                const categoryId = category.value;
+                categoriesToReturn.push(this.getCategory(categoryId));
+            }
+        }
+        return categoriesToReturn;
```

Comment:
Need to make the result unique, if a Category has two tags then it will be duplicated...

Reply:
A Category can have 2 tags with the same name?